korman/exporter/physics.py

```diff
@@ -77,6 +77,14 @@ def generate_physical(self, bo, so, bounds, name=None):
             physical.sceneNode = self._mgr.get_scene_node(bl=bo)
 
             getattr(self, "_export_{}".format(bounds))(bo, physical)
+            
+            if self._exporter().has_coordiface(bo):
+                if not physical.mass:
+                    # ...Since the object has a coordinate interface but isn't a kickable, it's likely going to be animated. In such case,
+                    # we must warn Plasma to have the collisions follow the object's coordinates as well.
+                    physical.mass = 1.
+                    simIface.setProperty(plSimulationInterface.kPinned, True)
+                    physical.setProperty(plSimulationInterface.kPinned, True)
```

Comment:
Ah, indeed, I missed that part in the animation modifier. It doesn't seem to handle parenting though- that must be the issue, since my test scene has an animated object parenting a physical one. Shoud I move this code to the animation modifier anyway ? To me always setting a mass to objects with a CoordinateInterface sounds safer.
I forgot kPinned only works on Complete Chronicles. It is however required, otherwise the object will start falling due to the nonzero weight (ugh, that sounds so stupid).
How do the ABM and MOUL versions of the engine handle this ? It seems MOUL uses the kPhysAnim flag, but it doesn't prevent the object from falling in CC.